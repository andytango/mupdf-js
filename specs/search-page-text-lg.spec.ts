import {
  forExamplePages,
  loadExampleFileSingleton,
} from "./helpers/example-file-lg";

const loadExampleFile = loadExampleFileSingleton();

describe("searchPageText with a large file", () => {
  forExamplePages("should search the page text", async (page) => {
    const { mupdf, doc } = await loadExampleFile();
    expect(mupdf.searchPageText(doc, page, "facebook", 10)).toMatchSnapshot();
  });
});
