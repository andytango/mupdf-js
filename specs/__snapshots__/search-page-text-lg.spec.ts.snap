// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`searchPageText with a large file Page [1 / 58] - should search the page text 1`] = `
[
  {
    "h": 12.391571,
    "isContinuation": true,
    "w": 34.493713,
    "x": 138.882996,
    "y": 113.477798,
  },
  {
    "h": 12.391571,
    "isContinuation": true,
    "w": 34.493713,
    "x": 129.358978,
    "y": 139.379776,
  },
  {
    "h": 12.391571,
    "isContinuation": true,
    "w": 34.493729,
    "x": 138.227982,
    "y": 152.331802,
  },
  {
    "h": 12.391571,
    "isContinuation": true,
    "w": 34.493729,
    "x": 117.903976,
    "y": 165.282791,
  },
  {
    "h": 12.391571,
    "isContinuation": true,
    "w": 38.247437,
    "x": 317.308014,
    "y": 205.248795,
  },
  {
    "h": 13.768311,
    "isContinuation": true,
    "w": 41.663605,
    "x": 198.695007,
    "y": 543.090027,
  },
  {
    "h": 13.768311,
    "isContinuation": true,
    "w": 41.663605,
    "x": 186.042007,
    "y": 590.911072,
  },
]
`;

exports[`searchPageText with a large file Page [2 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [3 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [4 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [5 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [6 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [7 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [8 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [9 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [10 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [11 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [12 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [13 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [14 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [15 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [16 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [17 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [18 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [19 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [20 / 58] - should search the page text 1`] = `
[
  {
    "h": 13.768311,
    "isContinuation": true,
    "w": 42.496857,
    "x": 153.964996,
    "y": 572.288025,
  },
]
`;

exports[`searchPageText with a large file Page [21 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [22 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [23 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [24 / 58] - should search the page text 1`] = `
[
  {
    "h": 13.768311,
    "isContinuation": true,
    "w": 41.122009,
    "x": 267.778992,
    "y": 517.436035,
  },
]
`;

exports[`searchPageText with a large file Page [25 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [26 / 58] - should search the page text 1`] = `
[
  {
    "h": 13.768311,
    "isContinuation": true,
    "w": 41.122009,
    "x": 286.888,
    "y": 345.726074,
  },
]
`;

exports[`searchPageText with a large file Page [27 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [28 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [29 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [30 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [31 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [32 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [33 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [34 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [35 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [36 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [37 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [38 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [39 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [40 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [41 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [42 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [43 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [44 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [45 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [46 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [47 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [48 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [49 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [50 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [51 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [52 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [53 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [54 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [55 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [56 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [57 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText with a large file Page [58 / 58] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;
